docs/_writing-tests/general-guidelines.md

```diff
@@ -112,6 +112,30 @@ either installed or to have specific metrics. As such, in most cases
 when a known font is needed Ahem should be used. In other cases,
 `@font-face` should be used.
 
+#### Be Conservative
+
+Sometimes, when authoring tests for one web platform feature, it may be
+convenient to use some unrelated feature (e.g. web browser APIs or ECMAScript
+language features). Reliance on such extraneous features tends to make tests
+more difficult to consume, so care should be taken when considering their use.
+Tests may only rely on on language features that are available in the latest
+version of all of the following browsers:
+
+- [Mozilla Firefox][mozilla-firefox]
+- [Google Chrome][google-chrome]
+- [Apple Safari][apple-safari]
+- [Microsoft Edge][microsoft-edge]
+
+For JavaScript code that is re-used across many tests (e.g. `testharness.js`
+and the files located in the directory named `common`), language features that
+were introduced in [ECMAScript 2015][es2015] or later should be avoided. This
+practice avoids introducing spanning test failures for consumers maintaining
```

Comment:
I was trying to communicate the severity of the failures--that they will span
multiple tests across all of WPT. But that detail is not particularly important
for this document, and the statement is no less accurate without the word, so
I'm happy to remove it.